CHANGELOG.md

```diff
@@ -4,19 +4,22 @@
 
 ### Breaking
 
+- (minor) Refactor the block schema and variations mechanism
```

Comment:
I won't call it a breaking, since the core is not using it yet, really. I would say it's a "part 2" instead, if not, we would have to leave it for 13...

Reply:
ok, not breaking then